rvbap.c

```diff
@@ -21,6 +21,8 @@ See copyright in file with name LICENSE.txt  */
 #include "m_pd.h"               /* you must include this - it contains the external object's link to pure data */
 #endif
 
+#define RVBAP_VERSION "rvbap v1.2 - (c) Olaf Matthes 2003-2007, based on vbap by Ville Pulkki"
```

Comment:
nitpick, but i guess `RVBAP_VERSION` should be just `1.2` and the rest of the string should not be part of *this* define.

Reply:
Eh, I'm not going to change these as they are only used internally. If we add a correct `VBAP_VERSION` define for a `VBAP_SETUP_MSG` define string, it's only used there anyway...

Reply:
I'm also letting the terrible formatting jumbled these sources go as well...